data/json/itemgroups/SUS/evac_shelter.json

```
@@ -3,16 +3,17 @@
     "id": "SUS_evac_shelter_locker",
     "type": "item_group",
     "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
-    "//2": "Each locker contains by spec one jacket, one blanket, one gas mask, a flashlight, a plastic bowl and utensils, a lighter, and two bottles of water.  Not every locker is fully stocked.",
+    "//2": "Each locker contains by spec one jacket, one blanket, one gas mask, a flashlight, a whistle, a plastic bowl and utensils, a lighter, and two bottles of water.  Not every locker is fully stocked.",
     "subtype": "collection",
     "entries": [
-      { "item": "evac_pamphlet", "prob": 99 },
-      { "item": "jacket_evac", "prob": 75 },
-      { "item": "emer_blanket", "prob": 75 },
+      { "item": "evac_pamphlet", "prob": 95 },
+      { "item": "jacket_evac", "prob": 72 },
+      { "item": "emer_blanket", "prob": 72 },
       { "item": "mask_gas", "prob": 2 },
-      { "item": "flashlight", "prob": 50 },
+      { "item": "flashlight", "prob": 48 },
       { "item": "light_disposable_cell", "prob": 10 },
       { "item": "lighter", "prob": 20 },
+      { "item": "whistle", "prob": 12 },
```

Whistles are common items. You don't need to have it only show up in 12% of the lockers.
```suggestion
      { "item": "whistle", "prob": 70 },
```

---

That was cool. How did you do that? All I had to do was press a button and your suggestion was added to the PR. Noice!